### PR TITLE
Rework README from end user point of view

### PR DESCRIPTION
**Pull request by kb2ma**
Make it clear how to prepare an image and run the migrator.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,103 @@
...
+*Our goal is to make it easy to migrate a computer to use balenaOS.* Let us know how it works for you. As an open source project, we're also happy to accept code and documentation contributions. See *Development* below.
 
-## Installation and Building
+## Getting Started
 
-Clone the migrator repository, install the node modules and build it as shown below.
+### Prepare image
+Migrator expects a pre-configured image for your device. You only need to perform these download and configure steps once for a fleet.
 
+Use [balenaCLI](https://docs.balena.io/reference/balena-cli/#os-download-type) or the balena [web site](https://www.balena.io/os#download-os) to download the image appropriate for your computer. Then configure it for your fleet. If you download from the web site, first unzip the compressed file.
```
Comment (alexgg) [inline README.md:20]: We need to specify a flasher image. Eventually we will most likely unify flasher and non flasher, but right now it's only supported for flasher images.

File: README.md [hunk 2]
```diff
@@ -1,49 +1,103 @@
...
+
+  > balena os configure balena-flasher.img --fleet MyFleet --version 2.113.12
 ```
-These commands generate a single-file executable `migrator.exe` in the `dist` directory.
 
-## Command reference documentation
-### Preconfigure Flasher Image
-You need a balenaOS image for your device type. See the downloads [page](https://www.balena.io/os) to retrieve the `<flasher-image-archive>` file. Run the commands below to configure the image. See the balenaCLI [documentation](https://docs.balena.io/reference/balena-cli/#os-configure-image) for details of the `os configure` command.
+### Run Migrator
+The command below prepares the computer for migration, and automatically reboots to execute it and launch balenaOS. You then should see the device appear with its fleet in your balenaCloud dashboard.
```
Comment (alexgg) [inline README.md:31]: I would add a warning that all contents of the disk will be erased and recommend to perform a backup of data beforehand.

File: README.md [hunk 3]
```diff
@@ -1,49 +1,103 @@
...
+
+```
+> git clone https://github.com/balena-os/migrator.git
+> cd migrator
+> npm install
+> npm run build
+> del node_modules\pkg\dictionary\drivelist.js
+> npm run pkg
+```
+These commands generate a single-file executable `migrator.exe` in the `dist` directory.
```
Comment (alexgg) [inline README.md:99]: I miss a list with future work that specifies that configuration migration and user data migration is not currenty supported but is on the roadmap.

File: README.md [hunk 4]
```diff
@@ -1,49 +1,103 @@
...
-> cd migrator
-> npm install
-> npm run build
-> npm run pkg
+  > balena os download generic-amd64 -o balena-flasher.img
+     Getting device operating system for generic-amd64
+     OS version not specified: using latest released version
+     balenaOS image version 2.113.12 downloaded successfully
+
+  > balena os configure balena-flasher.img --fleet MyFleet --version 2.113.12
```
Comment (alexgg) [inline README.md:27]: I would also mention that they should make sure balenaOS can run in their device and it appears in the dashboard. Maybe link to a Getting Started for Generic AMD64 as an example.

Also, a note that the device needs to have an ethernet and an DHCP server needs to be configured in the network is an important caveat until configuration migration is supported.

**Review by alexgg — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kb2ma) [reply README.md:20]: How can a user determine if their device accepts a flasher image? Can we just say that the device must boot from internal storage?

File: README.md (hunk 2 above)
Comment (kb2ma) [reply README.md:31]: That warning is at the top of the README. Should I add it here as well?

File: README.md (hunk 3 above)
Comment (kb2ma) [reply README.md:99]: How about if we link to the roadmap [page](https://balena.fider.io/posts/2/provide-tool-to-onboard-migrate-devices-already-deployed-in-the-field) for the migrator so it's all in one place?

**Comment by kb2ma** [comment]
Check out the updated introduction and Requirements, [here](https://github.com/balena-os/migrator/blob/357b2b92c30690b817e7c1bf0c10e30f876aad84/README.md).

File: README.md (hunk 1 above)
Comment (alexgg) [reply README.md:26]: As you know only x86_64 can run Windows, and both Generic GPT and MBR images are flasher images, so for the Windows migrator all images are flasher images.
When Linux come into play, we will need to deploy flasher images for all devices, or unify the flasher/non-flasher, whatever comes sooner.

Comment (kb2ma) [reply README.md:26]: Thanks for clarifying. According to [this doc](https://learn.microsoft.com/en-us/windows/arm/overview), Windows now runs on ARM as well.

Comment (kb2ma) [reply README.md:26]: I'm looking at the supported device types, and I'd like to be precise on which images are flasher images. Is it only Generic x86_64 (GPT) and Generic x86_64 (legacy MBR)? For example, is Intel NUC a flasher image? A user of that device type might expect to use that specific image.

File: README.md (hunk 2 above)
Comment (kb2ma) [reply README.md:37]: Adding second warning here.

File: README.md (hunk 3 above)
Comment (kb2ma) [reply README.md:105]: Added link to roadmap page.

File: README.md (hunk 4 above)
Comment (alexgg) [reply README.md:33]: @kb2ma echoing my opinion above, I don't think this README should delve into device specifics. That information will just be unmaintained and outdated. If needed I would modify specific device repositories to say that migration is supported.

Comment (kb2ma) [reply README.md:33]: I assume the only supported flasher images ATM are the two generic x86, which answers the question.

**Comment by kb2ma** [comment]
I self-certify!

**Pull request merged**